docs/product-user-guide/src/main/asciidoc/proc-business-central-settings-data-sources.adoc

```diff
@@ -0,0 +1,4 @@
+[[business_central_settings_data_sources]]
+= Data Sources
+
+This setting enables you to configure connections to databases. It is used internally with the application server on which Business Central is running. Alternatively, if you have connections to other servers you can configure the connections here. You can then use data from these servers in your dashboards.
```

Comment:
I'd rewrite this paragraph:

This perspective enables you to configure connections to external databases. In order to connect to external database you need an appropriate JDBC Driver available, which you can also add in this perspective. The data sources created in this perspective can be used to create SQL data sets in Data Sets (link to section on Data Sets).

-----------

To get a conceptual overview about how that fits within other workbench features you might want to check my [not-yet-outdated presentation](http://file.brq.redhat.com/~jhrcek/presentation/dashbuilder.html#(1)) about dashbuilder.